dtcw

```diff
@@ -163,7 +165,7 @@ EOF
 }
 
 print_version_info() {
-    echo "dtcw ${DTCW_VERSION} - ${DTCW_GIT_HASH}"
+    echo "dtcw ${DTCW_VERSION} (${os} ${arch}) - ${DTCW_GIT_HASH}"
```

Comment:
I'm not really happy about this change.
The `--version` output is also meant to be easy parsable by scripts (for example in the CI build).
Parsing for a version number on `dtcw 0.50 (Linux x86_64) - ##DTCW_GIT_HASH##` doesn't make is easier.
What about we change the output for the OS/architecture in a separate line?
```
dtcw 0.50 - ##DTCW_GIT_HASH##
docToolchain 2.2.1
OS/Arch: Linux/x86_64
```

Reply:
I will change this on all the scripts.

Reply:
And I was already wondering why the tests didn't fail. 
The reason was I dynamically parse the version from `--version` which now had the value `0.5.0 (Linux x84_64)`.

Reply:
OK. I did this change because a simple `./dtcw` command didn't output the version.
And I wanted to see on which system the script is running. I think this is important when users file issues.
Maybe we can modify it in such a way that all requirements are met?